.gitignore

```diff
@@ -17,3 +16,4 @@ ipl.rom
 !res/qoob_sx_13c_upgrade.elf
 
 .direnv
+/.vscode
```

Comment:
Not big on this. We can't add every editor/IDE's stuff in here. direnv is a soft exception to this because it's tooling that is explicitly supported in the repo.

You can achieve the same effect on your end by adding to `.git/info/exclude`, or [setting up a global excludes file](https://stackoverflow.com/a/7335487).

Reply:
Removed in the last rebase.